android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -763,15 +763,21 @@ public void setAuthorization(
                 username,
                 password
         );
-        consensusClient.setAuthorization(
+        blockClient.setAuthorization(
                 username,
                 password
         );
-        blockClient.setAuthorization(
+        untrustedClient.setAuthorization(
                 username,
                 password
         );
-        untrustedClient.setAuthorization(
+    }
+
+    /**
+     * Sets HTTP authorization username and password for consensus server requests.
+     */
+    public void setConsensusAuthorization(@NonNull String username, @NonNull String password) {
```

Comment:
Leaving a note that this codepath isn't under test (although I don't consider it PR-blocking).

Reply:
SG